src/openms/source/CONCEPT/QCMS2IdentificationRate.cpp

```diff
@@ -17,36 +17,37 @@ QCMS2IdentificationRate::~QCMS2IdentificationRate(){
 int QCMS2IdentificationRate::MS2IDRateidentifier_( MzTabFile& MzTabOutputFile, String out)
```

Comment:
- should be `MzTab& mztab` not `MzTabFile& MzTabOutputFile`.
 - String should be passed as `const &`
 - Method is public hence no underscore in name